R/plot_nhdplus.R

```diff
@@ -125,13 +128,18 @@ plot_nhdplus <- function(outlets = NULL, bbox = NULL, streamorder = NULL,
   # Work with cache data
   save <- FALSE
   fetch <- TRUE
-  if(!is.null(cache_data)) {
-    if(file.exists(cache_data)) {
-      pd <- readRDS(cache_data)
-      fetch <- FALSE
-    } else {
-      save <- TRUE
+  if(!isFALSE(cache_data)) {
+    if(!is.null(cache_data)) {
+      if(file.exists(cache_data)) {
+        pd <- readRDS(cache_data)
+        fetch <- FALSE
+      } else {
+        save <- TRUE
+      }
     }
+    cache_osm <- osm_cache_dir()
+  } else {
+    cache_osm <- file.path(tempdir(check = TRUE), "osm.cache")
```

Comment:
might be semantics - but this is a basemap rather than osm cache I think? OSM is one of many basemaps

Reply:
It's kind of both --- I'm using rosm for basemaps, so osm in in this context is indicating it's a cache for that package.